app/src/main/java/com/johnsondev/doboshacademyapp/MoviesAdapter.kt

```diff
@@ -1,61 +1,70 @@
+import com.johnsondev.doboshacademyapp.model.Movie
 
 class MoviesAdapter(
-    context: Context,
-    private val clickListener: OnRecyclerItemClicked
+        private val context: Context,
+        private val clickListener: OnRecyclerItemClicked
 ) : RecyclerView.Adapter<MovieViewHolder>() {
 
     private val inflater: LayoutInflater = LayoutInflater.from(context)
 
-    override fun getItemCount(): Int = Movie.movies.size
+    override fun getItemCount(): Int = MovieRepository.moviesList.size
 
-    fun getItem(position: Int): Movie = Movie.movies[position]
+    private fun getItem(position: Int): Movie = MovieRepository.moviesList[position]
 
     override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): MovieViewHolder {
         val itemView = inflater.inflate(R.layout.movie_rv_item, parent, false)
-        return MovieViewHolder(itemView)
+        return MovieViewHolder(itemView, context)
     }
 
     override fun onBindViewHolder(holder: MovieViewHolder, position: Int) {
         holder.bind(getItem(position))
         holder.itemView.setOnClickListener() {
-            clickListener.onClick(Movie.movies[position])
+            clickListener.onClick(MovieRepository.moviesList[position])
         }
     }
-
 }
 
-class MovieViewHolder(view: View) : RecyclerView.ViewHolder(view) {
+class MovieViewHolder(view: View, context: Context) : RecyclerView.ViewHolder(view) {
 
     private val name: TextView = view.findViewById(R.id.movie_name)
-    private val genre: TextView = view.findViewById(R.id.movie_genre)
+    private val genre: TextView = view.findViewById(R.id.movie_genres)
     private val reviews: TextView = view.findViewById(R.id.movie_reviews)
     private val rating: RatingBar = view.findViewById(R.id.movie_rating)
     private val time: TextView = view.findViewById(R.id.movie_time)
-    private val age: TextView = view.findViewById(R.id.movie_age)
+    private val age: TextView = view.findViewById(R.id.tv_age)
     private val movieImg: ImageView = view.findViewById(R.id.movie_img)
-    private val isFavorite: ImageView = view.findViewById(R.id.is_favorite_red_img)
+
+    private val strReviews: String = context.getString(R.string.reviews)
+    private val strMin: String = context.getString(R.string.min)
+    private val strPlus: String = context.getString(R.string.plus)
 
     fun bind(movie: Movie) {
-        name.text = movie.name
-        genre.text = movie.genre
-        reviews.text = "${movie.reviews} REVIEWS"
-        rating.progress = movie.rating
-        time.text = "${movie.time} MIN"
-        age.text = "${movie.age}+"
-        movieImg.setImageResource(movie.movieImg)
-        isFavorite.visibility = if (movie.isFavorite) View.VISIBLE else View.GONE
+        name.text = movie.title
+        genre.text = movie.genres.joinToString { it.name }
+        reviews.text = "${movie.reviewCount} $strReviews"
+        rating.progress = movie.rating * 2
+        time.text = "${movie.runningTime} $strMin"
```

Comment:
Check getString with multiple parameters. So you can make it:
```
context.getString(R.string.running_time, numberOfMinutes, minutes)
```

But actually you don't need even do such. Just have string with placeholder in strings:
```
<string id="running_time">%d mins</string>
```